store/sql_user_store.go

```diff
@@ -369,6 +372,28 @@ func (us SqlUserStore) GetByEmail(teamId string, email string) StoreChannel {
 	return storeChannel
 }
 
+func (us SqlUserStore) GetByAuth(teamId string, authData string, authService string) StoreChannel {
```

Comment:
Since this is only called from the oauth path I don't think we would figure out it's broke until someone tried to run oauth on postgres.